src/test/messageHandlers/JSONHandlerTest.java

```diff
@@ -0,0 +1,39 @@
+package messageHandlers;
```

Comment:
This is not the same package as the SRC file. Although I think this name better fits the subject.